draft-ietf-quic-transport.md

```diff
@@ -2609,15 +2605,22 @@ details.
 
 ## Coalescing Packets {#packet-coalesce}
 
-A sender can coalesce multiple QUIC packets into one UDP datagram.  This can
-reduce the number of UDP datagrams needed to complete the cryptographic
-handshake and starting sending data.  Receivers MUST be able to process
-coalesced packets.
+Initial ({{packet-initial}}), 0-RTT ({{packet-0rtt}}), and Handshake
+({{packet-handshake}}) packets contain a Length field, which determines the end
```

Comment:
I think we might want to either rename the Length field, or at least clarify that it's in a different place based on the Long Header Type.